src/foobar/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from djmoney.forms.fields import MoneyField
+from foobar.wallet import api
+from django.utils.translation import ugettext as _
+
+
+class CorrectionForm(forms.Form):
+    balance = MoneyField(label='Balance', min_value=0)
+    comment = forms.CharField(label='Comment', max_length=250, required=False)
```

Comment:
The max length of the comment field should reflect the max length of the corresponding database field.